### PR TITLE
Add terraform 0.13 support

### PR DESCRIPTION
**Pull request by pontostroy**
Allow use terraform-null-label with terraform 0.13.

We are using terraform-null-label and terraform 0.13, and would like to see these changes in upstream.

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = "~> 0.13.0"
```
Comment (osterman) [inline versions.tf:2]: We want to support both `0.12` and `0.13` for some time.

```suggestion
  required_version = ">= 0.12.0, < 0.14.0"
```

**Comment by davidkarlsen** [comment]
Fixes #96

**Comment by davidkarlsen** [comment]
Anything else needed here @osterman ?

**Comment by osterman** [comment]
@davidkarlsen looks like chatops broke on this repo. fixed and retesting.

**Pull request merged**